src/app/calendar/calendar.cpp

```diff
@@ -1 +1,5 @@
-#include "calendar.hpp"
\ No newline at end of file
+#include "calendar.hpp"
+
+void Calendar::createCalendar(){
+    Calendar calendar{};
```

Comment:
I think we don't need to have the whole callendar. We need proper calculate of short period of time like month or week.